meta-mender-demo/recipes-mender/mender-client/mender-client_%.bbappend

```diff
@@ -1,16 +1,10 @@
-FILESEXTRAPATHS_prepend_mender-client-install := "${THISDIR}/files:"
-
-SRC_URI_append_mender-client-install = " file://server.crt"
-
 MENDER_UPDATE_POLL_INTERVAL_SECONDS_mender-client-install = "5"
 MENDER_INVENTORY_POLL_INTERVAL_SECONDS_mender-client-install = "5"
 MENDER_RETRY_POLL_INTERVAL_SECONDS_mender-client-install = "30"
 
-MENDER_CERT_LOCATION_mender-client-install ?= "${docdir}/mender-client/examples/demo.crt"
-
-# We need this because the certificate will automatically end up in the
-# mender-doc package when placed in ${docdir}.
-RDEPENDS_${PN}_append_mender-client-install = " ${PN}-doc"
+# Depend on mender-server-certificate, which on demo layer installs the demo cert
+DEPENDS_append = " mender-server-certificate"
+RDEPENDS_${PN} = "mender-server-certificate"
```

Comment:
`.bbappend` files should always use `_append`, for both entries.

Reply:
Yes, thanks!